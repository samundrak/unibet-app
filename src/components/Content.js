import React, { Fragment } from 'react';
import Carousel from './Carousel/index';
import ScoreBoard from './Scoreboard';

const Content = () => {
  return (
    <Fragment>
      <h1>Live matches</h1>
      <p className="preamble">
        Here is a list of matches that are live right now.
      </p>
      <aside>
        <h2>Live betting</h2>
        <p>
          Place your bets as the action unfolds. We offer a wide selection of
          live betting events and you can place both single and combination
          bets.
        </p>
        <p>
          You will be able to see an in-play scoreboard with the current result
          and match stats, while on selected events you will also be able to
          watch the action live with Unibet TV on the desktop site.
        </p>
      </aside>
      <div id="live-matches">
        <Carousel>
          <ScoreBoard />
          <ScoreBoard />
        </Carousel>
      </div>
    </Fragment>
  );
};
export default Content;
