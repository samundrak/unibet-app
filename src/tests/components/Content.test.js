import React from 'react';
import { render } from 'react-dom';
import Content from '../../components/Content';

describe('test content', () => {
  it('should test snapshot', () => {
    const container = document.createElement('div');
    render(
      <Content
        totalRecords={1}
        loading={false}
        records={new Map([[1, { event: { score: {} } }]])}
        wasErrorFetchingLiveScore={false}
      />,
      container,
    );
    expect(container.innerHTML).toMatchSnapshot();
  });
});
