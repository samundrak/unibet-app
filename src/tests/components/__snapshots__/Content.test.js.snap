// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test content should test snapshot 1`] = `"<h1>Live matches</h1><p class=\\"preamble\\">Here is a list of matches that are live right now.</p><aside><h2>Live betting</h2><p>Place your bets as the action unfolds. We offer a wide selection of live betting events and you can place both single and combination bets.</p><p>You will be able to see an in-play scoreboard with the current result and match stats, while on selected events you will also be able to watch the action live with Unibet TV on the desktop site.</p></aside><div id=\\"live-matches\\"><div width=\\"600px\\" height=\\"300px\\" class=\\"sc-bdVaJa hemWUb\\"><div class=\\"sc-htpNat bMtrgk\\"><div class=\\"sc-gzVnrw gWtOMw\\"><div class=\\"sc-htoDjs dZmTQw\\"><div class=\\"sc-dnqmqq hNfzMe\\"></div>-<div class=\\"sc-dnqmqq hNfzMe\\"></div></div><div class=\\"sc-gZMcBi hdRYkI\\"><img src=\\"/icons/default.png\\"><div class=\\"sc-iwsKbI bHJQaO\\"></div>-<div class=\\"sc-iwsKbI bHJQaO\\"> </div></div><div class=\\"sc-gqjmRU iXpBVF\\">Invalid Date</div><div class=\\"sc-VigVT dwVcPI\\"><button class=\\"sc-jTzLTM eJrpoN\\">Place a Bet</button></div></div></div><div class=\\"sc-htpNat bMtrgk\\"><div class=\\"sc-gzVnrw gWtOMw\\"><div class=\\"sc-htoDjs dZmTQw\\"><div class=\\"sc-dnqmqq hNfzMe\\"></div>-<div class=\\"sc-dnqmqq hNfzMe\\"></div></div><div class=\\"sc-gZMcBi hdRYkI\\"><img src=\\"/icons/default.png\\"><div class=\\"sc-iwsKbI bHJQaO\\"></div>-<div class=\\"sc-iwsKbI bHJQaO\\"> </div></div><div class=\\"sc-gqjmRU iXpBVF\\">Invalid Date</div><div class=\\"sc-VigVT dwVcPI\\"><button class=\\"sc-jTzLTM eJrpoN\\">Place a Bet</button></div></div></div><div class=\\"sc-bxivhb keacqw\\"></div></div></div>"`;
